test/index.js

```diff
@@ -513,7 +513,9 @@ test('equals', function() {
   eq(Z.equals(Math.sin, Math.cos), false);
   eq(Z.equals(Identity(Identity(Identity(0))), Identity(Identity(Identity(0)))), true);
   eq(Z.equals(Identity(Identity(Identity(0))), Identity(Identity(Identity(1)))), false);
-  eq(Z.equals({'@@type': 'my-package/Quux'}, {'@@type': 'my-package/Quux'}), false);
+  eq(Z.equals({'@@type': 'my-package/Quux'}, {'@@type': 'my-package/Quux'}), true);
```

Comment:
so here `Z.equals` delegates to `equals` from Setoid instance of Object even if this objects had `fl.equals` defined